CMakeLists.txt

```diff
@@ -381,7 +381,7 @@ if(WIN32)
   
   # Win64 specific Thrift
   set(THRIFT_INCLUDE_DIR "${WINDOWS_DEP_DIR}/thrift-dev/local/include")
-  set(THRIFT_LIBRARY "${WINDOWS_DEP_DIR}/thrift-dev/local/lib/thriftmd.lib")
+  set(thrift_library "${WINDOWS_DEP_DIR}/thrift-dev/local/lib/thriftmd.lib")
```

Comment:
I believe this: https://github.com/trailofbits/osquery/blob/master/osquery/CMakeLists.txt#L92